datalad/support/sshconnector.py

```diff
@@ -294,6 +294,11 @@ def _get_scp_command_spec(self, recursive, preserve_attrs):
     def put(self, source, destination, recursive=False, preserve_attrs=False):
         """Copies source file/folder to destination on the remote.
 
+        Note: this method perform escaping of filenames to an extend that
+        moderately weird ones should work (spaces, quotes, pipes, other
+        characters with special shell meaning), but more complicated cases
+        might require appropriate external preprocessing of filenames.
+
```

Comment:
My understanding is that the upstream change only concerns remote => local, so making this change for put() isn't strictly necessary.  Though for consistency perhaps it still makes sense for put() and get() to have consistent file name washing.